security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem and therefore also at the heart of the SSWG. This page intends to give clear and easy to follow information to users and developers of packages listed on the SSWG's package index.
+
+If you find any information to be missing, unclear, or misleading, please contact any member of the SSWG or create a post in the [server area](https://forums.swift.org/c/server/serverdev/14) of the Swift Forums.
+
+## I found a security vulnerability, what should I do?
+
+If you found a security vulnerability, first of all thank you for caring about our ecosystem. We rely on the community to keeping us all as safe and secure as we can (at least when it comes to software).
+
+The SSWG only takes a secondary role in the security process. The SSWG makes sure that listed projects do have a security process and that they follow their own process but the SSWG is typically not the first point of contact. Please don't read this as sending you away though, if at any point in the process you feel like you don't know where to report, you don't get a response, or face any other issues, please do contact the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org) (this can only be seen by the SSWG members and the Swift core team).
+
+From packages on the SSWG's package index, it is required to list a clear point of contact on the project's main information page. Typically that is the `README.md` file in the project's GitHub/GitLab/... repository. It is also required that the project list the exact steps on how to report a vulnerability. Please follow these steps, the SSWG will be notified by the project directly. If you can't find any required information, you don't get a response from the project's maintainers, or you a are unsure that the vulnerability is treated with the appropriate care and urgency, please do contact the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org).
```

Comment:
Grammar/phrasing nits:
```suggestion
Packages on the SSWG's package index are required to list a clear point of contact on the project's main information page. Most often, this will be the `README.md` file at the root of the project's GitHub/GitLab/etc. repository. Projects are also required to list exact steps for reporting a vulnerability. Please follow these steps; the SSWG will be notified by the project directly. If you can't find any required information, you don't get a response from the project's maintainers, or you feel that the vulnerability is not being treated with the appropriate care and urgency, please reach out to the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org).
```